pkg/services/featuremgmt/toggles_gen.json

```diff
@@ -2223,11 +2223,25 @@
         "hideFromDocs": true
       }
     },
+    {
+      "metadata": {
+        "name": "preserveDashboardStateWhenNavigating",
+        "resourceVersion": "1716303823534",
+        "creationTimestamp": "2024-05-21T15:03:43Z"
+      },
+      "spec": {
+        "description": "Enables possibility to preserve dashboard variables and time range when navigating between dashboards",
+        "stage": "experimental",
+        "codeowner": "@grafana/dashboards-squad",
+        "hideFromAdminPage": true,
+        "hideFromDocs": true
+      }
+    },
     {
       "metadata": {
         "name": "datasourceProxyDisableRBAC",
-        "resourceVersion": "1715889033198",
-        "creationTimestamp": "2024-05-16T19:50:33Z"
+        "resourceVersion": "1716303823534",
+        "creationTimestamp": "2024-05-21T15:03:43Z"
```

Comment:
minor thing and i'm sure it probably doesn't matter, but i don't think there should be any changes to this unrelated feature toggle 🤔

Reply:
merge issue I think